spk/tt-rss/Makefile

```diff
@@ -11,20 +11,17 @@ DESCRIPTION = Tiny Tiny RSS is an open source web-based news feed \(RSS/Atom\) r
 DESCRIPTION_FRE = Tiny Tiny RSS est un agrégateur et lecteur web de flux RSS et Atom open source conçu pour vous permettre de lire des nouvelles de n\'importe quel endroit, tout en se sentant aussi proche d\'une application de bureau que possible.
 ADMIN_URL = /tt-rss/
 DISPLAY_NAME = Tiny Tiny RSS
-CHANGELOG = "1. Update Tiny Tiny RSS to Git revision c30b24d09f (20221230) <br /> 2. Fixed broken DSM 7 initial installation procedure. <br /> 3. Upgrade PHP version to 7.4"
+CHANGELOG = "Update Tiny Tiny RSS to Git revision afd04d141c (20230828)"
 
 HOMEPAGE   = https://tt-rss.org/
 LICENSE    = GPL
 
```

Comment:
I noted in your build log for DSM 6 the following error:
```
  ===>  NOT creating /github/workspace/spk/tt-rss/work-dsm6/conf/privilege
  ===>  (privilege) DSM <= 6 and SERVICE_USER undefined
```

I would suggest that you add code similar to this to resolve this issue and allow the user `sc-tt-rss` to be created under DSM 6: https://github.com/SynoCommunity/spksrc/pull/5916/commits/cb744189368cbc21791bbc9e22c95d048cad29dc